examples/src/main/java/com/example/ftclibexamples/FlywheelSample.java

```diff
@@ -0,0 +1,63 @@
+package com.example.ftclibexamples;
+
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.arcrobotics.ftclib.gamepad.GamepadKeys;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorGroup;
+import com.qualcomm.hardware.lynx.LynxModule;
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import java.util.List;
+
+/**
+ * A sample opmode for a flywheel with two motors
+ * that are linked mechanically.
+ */
+@TeleOp
+@Disabled
+public class FlywheelSample extends LinearOpMode {
+
+    private GamepadEx toolOp;
+    private MotorGroup flywheel;
+
+    public static double kP = 20;
+    public static double kV = 0.7;
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        toolOp = new GamepadEx(gamepad2);
+
+        flywheel = new MotorGroup(
+                new Motor(hardwareMap, "flywheel_left", Motor.GoBILDA.BARE),
+                new Motor(hardwareMap, "flywheel_right", Motor.GoBILDA.BARE)
+        );
+
+        flywheel.setRunMode(Motor.RunMode.VelocityControl);
+        flywheel.setVeloCoefficients(kP, 0, 0);
+        flywheel.setFeedforwardCoefficients(0, kV);
+
+        List<LynxModule> hubs = hardwareMap.getAll(LynxModule.class);
+        hubs.forEach(hub -> hub.setBulkCachingMode(LynxModule.BulkCachingMode.MANUAL));
```

Comment:
I think overall we need more verbose commenting

Reply:
Agreed. Sample here needs a lot more comments.